app/src/main/java/com/babylon/wallet/android/presentation/accessfactorsources/deriveaccounts/DeriveAccountsViewModel.kt

```diff
@@ -206,15 +212,15 @@ class DeriveAccountsViewModel @Inject constructor(
                 accessFactorSourcesIOHandler.setOutput(
                     output = AccessFactorSourcesOutput.DerivedAccountsWithNextDerivationPath(
                         derivedAccounts = derivedAccounts,
-                        nextDerivationPathOffset = indicesToScan.last() + 1u
+                        nextDerivationPathIndex = HdPathComponent.init(indicesToScan.last().indexInGlobalKeySpace + 1u)
```

Comment:
same here, can you spell out `globalKeySpace = ` ?

Reply:
Will do